app/helpers/profile_helper.rb

```diff
@@ -0,0 +1,19 @@
+#   Copyright (c) 2010-2011, Diaspora Inc.  This file is
+#   licensed under the Affero General Public License version 3 or later.  See
+#   the COPYRIGHT file.
+
+module ProfileHelper
+  def upper_limit_date_of_birth
+    minimum_year = AppConfig.settings.terms.minimum_age.get
+    if !(minimum_year)
+      minimum_year = 13
+    else
+      minimum_year = minimum_year.to_i
+    end
```

Comment:
Thank you for the explanation. I wasn't aware of the convention followed in diaspora.
I have updated the pull request and squashed the commits.